### PR TITLE
[Image] use img tag

### LINKED ISSUE #892: [Image] use img tag

### PR DESCRIPTION
**Pull request by MateusStanki**
## Closes Issues

* Closes #892 

### Reviewer 1

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### Reviewer 2

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### CODE REVIEW

File: packages/image/src/Image.story.js [hunk 1]
```diff
@@ -69,14 +58,43 @@ storiesOf("Image", module)
       <Image src={imgSrc} badge size={badgeSizes.LARGE} />
     </div>
   ))
-  .add("fit (cover)", () => (
+  .add("custom dimensions", () => (
+    <Image src={imgSrc} width="800px" height="400px" />
+  ))
+  .add("fit (default/cover) - horizontal image", () => (
```
Comment (alcferreira) [inline packages/image/src/Image.story.js]: What about changing these stories to group a set of props? Something similar to button and tooltip stories.

I guess we could have 
- 1 story covering thumbnail and its sizes
- 1 story covering badge and its sizes
- 1 story for fit and all its cases
- Other stories that don't fit the above cases

File: packages/image/src/constants/fitModes.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const fitModes = {
-  COVER: "cover",
   CONTAIN: "contain",
+  FILL: "fill",
+  COVER: "cover",
```
Comment (vNakamura) [inline packages/image/src/constants/fitModes.js]: As commented in the Readme, there's another possible value: `scale-down`.
Please add it here and include a small image to the stories.

File: packages/image/src/constants/fitModes.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
+  "SCALE-DOWN": "scale-down",
```
Comment (alcferreira) [inline packages/image/src/constants/fitModes.js]:
```suggestion
  "SCALE_DOWN": "scale-down",
```

File: packages/image/README.md [hunk 4]
```diff
@@ -37,16 +37,16 @@ document.body.appendChild(root);
...
-| fit      | how should the image fit the size (same options that the background-scale css attribute)    | string                                         |
-| css      | custom css properties                                                                       | object                                         |
+| Property     | Description                                                                                 | Type    |
+| ------------ | ------------------------------------------------------------------------------------------- | ------- |
+| src          | **required**, the URL of the image                                                          | string  |
+| badge        | make the image a circle                                                                     | boolean |
+| borderRadius | the border radius of the image                                                              | string  |
+| css          | custom css properties                                                                       | object  |
+| fit          | how should the image fit the size (same options that the background-scale css attribute)    | string  |
```
[the remark this answers is not in the document]
Comment (vNakamura) [reply packages/image/README.md]:
```suggestion
| fit          | how should the image fit the size (same options that the object-fit css attribute)    | string  |
```

**Pull request merged**